jinahub/crafters/ImageNormalizer/normalizer.py

```diff
@@ -24,7 +26,19 @@ def __init__(
         *args,
         **kwargs,
     ):
-        """Set Constructor."""
+        """
+        `ImageNormalizer` resizes, crops and normalizes images stored in Document blobs.
```

Comment:
Is it necessary to have the class description in the init docstring?

Reply:
no it's not necessary
I made research on what is the best way to write the docstrings
It's not clear if an overview of what a constructor does can be included under the constructor.
Currently, we dont' display the docstring, it's just used to extract the description of the parameters

Reply:
So for convenience, I'll keep only parameters in the docstring

Reply:
done